### PR TITLE
Changed scaling factor in betweenness.py (#2709)

### PR DESCRIPTION
**Pull request by regstrtn**
This is my first ever attempt at fixing a bug. The issue #2709 came across as simpler than the rest. 
I changed the _rescale function to take endpoints as an input parameter. 
If endpoints is true: 
scale = 1/(n*(n-1))
else:
scale = 1/((n-1)*(n-2))
Please review

### CODE REVIEW

File: networkx/algorithms/centrality/betweenness.py [hunk 1]
```diff
@@ -320,12 +320,15 @@ def _accumulate_edges(betweenness, S, P, sigma, s):
...
 
-def _rescale(betweenness, n, normalized, directed=False, k=None):
+def _rescale(betweenness, n, normalized, directed=False, k=None, endpoints=False):
     if normalized:
         if n <= 2:
             scale = None  # no normalization b=0 for all nodes
         else:
-            scale = 1.0 / ((n - 1) * (n - 2))
+            if endpoints:
+                scale = 1.0 / ((n) * (n - 1)) # If endpoints are included, change the scale factor to include these nodes
```
Comment (jfinkels) [inline networkx/algorithms/centrality/betweenness.py]: Let's move the comment to the line directly above, so that this line doesn't become too long (i.e., 80 characters or more).

Comment (jfinkels) [inline networkx/algorithms/centrality/betweenness.py]: Let's place the line `from __future__ import division` at the head of the file but after the module-level docstring (i.e. just above the line `from heapq import heappush, heappop`). See https://www.python.org/dev/peps/pep-0238/. I like to remove the `.0` in this case because it is an unnecessary artifact, but it's more of a matter of preference, so you don't have to do it if you prefer to minimize changes here.

The same comment applies to the line in the `else` branch as well.

File: networkx/algorithms/centrality/betweenness.py [hunk 2]
```diff
@@ -320,12 +320,15 @@ def _accumulate_edges(betweenness, S, P, sigma, s):
     return betweenness
 
 
-def _rescale(betweenness, n, normalized, directed=False, k=None):
+def _rescale(betweenness, n, normalized, directed=False, k=None, endpoints=False):
     if normalized:
         if n <= 2:
```
Comment (jfinkels) [inline networkx/algorithms/centrality/betweenness.py]: Does the `n == 2` case matter when `endpoints` is `True`?

**Comment by dschult** [comment]
This looks close to ready.  Can you make your changes consistent with PEP8 style guides. Search for PEP8 for the guidelines. You can install a pep8 checking program using ```pip install pep8``` to check your code.

The other nice features would be to add a test of the rescaling, and to check make the check for ```n==2``` case provide non-zero scaled values when ```endpoints is True```.

Thanks!
